grails-app/views/job/_jobTable.gsp

```diff
@@ -18,16 +18,19 @@
                             class="fas fa-chart-bar"></i></a>
                 </g:if>
             </div>
-
             <div class="jobNameContainer"><a
                     href="${createLink(action: 'edit', id: job.id)}"
-                    class="jobName ${job.active == false ? 'inactiveJob' : ''}">
+                    class="jobName ${job.active == false ? 'inactiveJob' : ''} ${(job.jobStatistic == null ? '' :
+                            (job.jobStatistic.getJobStatusLast5CssClass() == 'job-status-error' ? 'jobName-status-error' : (job.jobStatistic.getJobStatusLast5CssClass() == 'job-status-warn' ? 'jobName-status-warn' : ''))
```

Comment:
Why don't you simply reuse the same class and use a scoped CSS selector to apply the color? Like
```
.jobName.job-status-error {
  color: @brand-danger;
}
```
You will save quite some code here.

Reply:
You are completely right! I did not think of that and will change it as soon as possible!